qiita_db/handlers/prep_template.py

```diff
@@ -0,0 +1,65 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from tornado.web import HTTPError
+
+import qiita_db as qdb
+from .oauth2 import OauthBaseHandler, authenticate_oauth
+
+
+def _get_prep_template(p_id):
+    """Returns the prep template with the given id if it exists
```

Comment:
In previous code we normally use pid but up to you. Perhaps, we should create a formal dictionary ...